src/arena.rs

```diff
@@ -0,0 +1,161 @@
+use std::collections::HashMap;
```

Comment:
Definitely see if things still work with Mateo's change

Reply:
That's my bad, I did check it all and apparently forgot to delete the actual class in my commit